docs/policy/software-release.md

```diff
@@ -14,20 +14,28 @@ The Software Team maintains the following Yum repositories:
 1.  **osg-testing**: This is where software goes when it is ready for wide-spread testing, including upstream release
     candidates
 1.  **osg-prerelease**: This is where software goes just before being released, for final verification.
-1.  **osg-rolling**: This is where software goes before being included in a point release. Intended for end-users.
 1.  **osg-release**: This is the official, production release of the software stack.
     This is the main repository for end-users.
 1.  **osg-contrib**: This is where software goes that is not officially supported by the OSG Software Team,
     but we provide as a convenience for software our users might find useful.
 
-We also create a repository per release, called **osg-release-VERSION** (such as osg-release-3.0.4).
-This is intended mostly for testing purposes, though users may occasionally find it useful.
-
 Occasionally there may be other repositories for specific short-term purposes.
 
+!!! note
+    **osg-rolling** and **osg-release-VERSION** are only present in the OSG 3.5 series.
+
+1.  **osg-rolling**: This is where software goes before being included in a point release. Intended for end-users.
+    In OSG 3.5, software goes into **osg-rolling** when it is put into **osg-prerelease**.
+
+We also create a repository per release, called **osg-release-VERSION** (such as osg-release-3.5.4).
+This is intended mostly for testing purposes, though users may occasionally find it useful.
+
 Version Numbers
 ---------------
 
+!!! note "OSG 3.5 Only"
+    This version number scheme is only applicable to the OSG 3.5 series.
+
```

Comment:
Looking at this section, maybe we should make this a sub-section and add a separate `3.6+` section that just says that the version numbers match the release series?

Reply:
Updated as requested